Makefile.toml

```diff
@@ -229,20 +229,41 @@ args = [
     "test",
     "--features",
     "${CARGO_FEATURES_TEST}",
+    "--",
+    "--test-threads=4",    
+    # "--nocapture",
 ]
 
-[tasks.test-flow]
+[tasks.build-aurora-eth-connector]
+category = "Build"
+script = '''
+AURORA_ETH_CONNECTOR_DIR="engine-tests-connector/etc/aurora-eth-connector/"
+AURORA_ETH_CONNECTOR_ETC="engine-tests-connector/etc/"
+AURORA_ETH_CONNECTOR_REPO_NAME="aurora-eth-connector"
+AURORA_ETH_CONNECTOR_REPO="https://github.com/aurora-is-near/"$AURORA_ETH_CONNECTOR_REPO_NAME
+if [ -d $AURORA_ETH_CONNECTOR_DIR ]; then
+  cd $AURORA_ETH_CONNECTOR_DIR
+  echo "Pull Aurora Eth-Connector repo"
+  git pull origin master
+else
+  mkdir $AURORA_ETH_CONNECTOR_ETC || true
+  cd $AURORA_ETH_CONNECTOR_ETC
+  echo "Clone Aurora Eth-Connector repo: "$AURORA_ETH_CONNECTOR_REPO
+  git clone --depth 1 $AURORA_ETH_CONNECTOR_REPO
+  cd $AURORA_ETH_CONNECTOR_REPO_NAME
+fi
+cargo make --profile mainnet build-test
```

Comment:
From `Cargo.toml`:
```
# Currently feature `mainnet` used only for 
# Makefile.toml profiles to indicate basic features
```
Keep in mind, currently, there is no default feature. Probably it can be converted just to default.